src/test/java/space/obminyashka/items_exchange/rest/UserControllerIntegrationTest.java

```diff
@@ -152,13 +161,29 @@ private static Stream<Arguments> listInvalidEmail() {
     @Test
     @WithMockUser
     void resetUserPassword_whenPasswordResetSuccessfully_shouldSendMessage() throws Exception {
+        when(userService.findByUsernameOrEmail(any())).thenReturn(Optional.of(new User()));
+        when(userService.existsByEmail(any())).thenReturn(true);
         var validatedEmailRequest = new ValidatedEmailRequest("email@gmail.com");
 
-        MvcResult mvcResult = sendDtoAndGetMvcResult(post(USER_SERVICE_RESET_PASSWORD), validatedEmailRequest, status().isOk());
+        MvcResult mvcResult = sendDtoAndGetMvcResult(post(USER_SERVICE_RESET_PASSWORD).header(HttpHeaders.HOST, DOMAIN_URL), validatedEmailRequest, status().isOk());
```

Comment:
Please keep lines in 180 characters length

Reply:
++